src/nifake/unit_tests/test_grpc.py

```diff
@@ -58,8 +58,8 @@ def __init__(self):
                     error_func = _mock_helper.MockFunctionCallError(f)
                     setattr(self, f, MagicMock(spec_set=self._sample_func, side_effect=error_func))
 
-        def __call__(self, session_options):
-            self._session_options = session_options
```

Comment:
I'm having trouble understanding this change.

Reply:
In Dan's PR, he [changed this](https://github.com/ni/nimi-python/commit/380fb5131b19859fe794b016f8a7da79246de6d2#diff-899f0ea835ab662dd4436c04e6f4de2ccc24050445a2871ef3bda7256bcee85b) from grpc_channel to session_options, as he started (but never finished) updating the tests. But it was an incorrect change; the Stub takes a grpc_channel. His PR was merged while tests were still failing. It was failing on line 86. So here I'm reverting that, and now the tests pass.

Reply:
So I guess you're creating self._grpc_channel (which normally doesn't exist in the class object) in order to make assertions about it later?

Reply:
Correct